examples/misc_compass.html

```diff
@@ -0,0 +1,131 @@
+
+
+        <div id="viewerDiv"></div>
+
+        <!-- Import iTowns source code -->
+        <script src="../dist/itowns.js"></script>
+        <script src="../dist/debug.js"></script>
+        <!-- Import iTowns LoadingScreen and GuiTools pluggins -->
+        <script src="js/GUI/LoadingScreen.js"></script>
+        <script src="js/GUI/GuiTools.js"></script>
+
+        <script type="text/javascript">
+
+
+
+            // ---------- CREATE A GlobeView FOR SUPPORTING DATA VISUALIZATION : ----------
+
+            // Define camera initial position
+            const placement = {
+                coord: new itowns.Coordinates('EPSG:4326', 2.351323, 48.856712),
+                range: 6000,
+                heading: 45,
+                tilt: 50,
+            }
+
+            // `viewerDiv` contains iTowns' rendering area (`<canvas>`)
+            const viewerDiv = document.getElementById('viewerDiv');
+
+            // Create a GlobeView
+            const view = new itowns.GlobeView(viewerDiv, placement);
+
+            // Setup loading screen and debug menu
+            setupLoadingScreen(viewerDiv, view);
+            const debugMenu = new GuiTools('menuDiv', view);
+
+
+
+            // ---------- DISPLAY CONTEXTUAL DATA : ----------
+
+            // Add one imagery layer to the scene. This layer's properties are defined in a json file, but it could be
+            // defined as a plain js object. See `Layer` documentation for more info.
+            itowns.Fetcher.json('layers/JSONLayers/Ortho.json').then((config) => {
+                config.source = new itowns.WMTSSource(config.source);
+                view.addLayer(
+                    new itowns.ColorLayer(config.id, config),
+                ).then(debugMenu.addLayerGUI.bind(debugMenu));
+            });
+
+            // Add two elevaion layers, each with a different level of detail. Here again, each layer's properties are
+            // defined in a json file.
+            function addElevationLayerFromConfig(config) {
+                config.source = new itowns.WMTSSource(config.source);
+                view.addLayer(
+                    new itowns.ElevationLayer(config.id, config),
+                ).then(debugMenu.addLayerGUI.bind(debugMenu));
+            }
+            itowns.Fetcher.json('layers/JSONLayers/WORLD_DTM.json').then(addElevationLayerFromConfig);
+            itowns.Fetcher.json('layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
+
+
+
+            // ---------- ROTATE THE COMPASS WHEN MOVING THE VIEW CAMERA : ----------
+
+            const compass = document.getElementById('compass');
+
+            // Update compass rotation when the camera is moved.
+            view.addEventListener(itowns.VIEW_EVENTS.CAMERA_MOVED, (event) => {
+                compass.style.transform = `
+                    rotate3d(1, 0, 0, ${90 - event.tilt}deg)
+                    rotate3d(0, 0, 1, ${-event.heading}deg)
+                `;
+            });
```

Comment:
could you try with this modification:
```js
 view.addEventListener(itowns.VIEW_EVENTS.CAMERA_MOVED, (event) => {
     compass.style.transform = `
            rotate3d(1, 0, 0, ${90 - event.tilt - view.camera.camera3D.fov}deg)
            rotate3d(0, 0, 1, ${-event.heading}deg)`;
});
```

Reply:
Doing so will make the compass appear a bit tilted when in top view, as seen in the following screenshot.

![90degree](https://user-images.githubusercontent.com/73115044/148387830-a2efbb22-4a6b-4f9a-a39a-41fc3b7b13cb.png)

Is there a behavior of the compass you wish to change doing so ?